src/common/store/review-manifest.store.js

```diff
@@ -85,17 +85,17 @@ export const useReviewManifestJson = () => {
     georeference: {
       lat: anchorPoint.coordinates[1],
       lon: anchorPoint.coordinates[0],
-      angle: anchorPoint.angle,
+      angle: fixAngleForManifest(anchorPoint.angle),
     },
     featureClasses: layers
-      .filter(layer => !layer.isDraft)
-      .map(layer => {
+      .filter(layer => !layer?.isDraft)
+      .map((layer = {}) => {
         const featureClass = {
           featureClassName: layer.name,
           dwgLayers: layer.value,
         };
 
-        const props = layer.props.filter(prop => !prop.isDraft);
+        const props = (layer.props || []).filter(prop => !prop?.isDraft);
```

Comment:
It's hard to comprehend cause this is massive PR but it feels a bit odd that you have to add default values in so many places. Perhaps it's better to validate data once somewhere at the top level, rather than adding default values, checks and optional chainings that often